bucket/safebrowser.json

```
@@ -0,0 +1,30 @@
+{
+	"version": "0.12.0",
+	"description": "SAFE Browser Application",
+	"homepage": "https://github.com/maidsafe/safe_browser",
+	"license": "BSD-3-Clause|MIT",
+	"url": "https://github.com/maidsafe/safe_browser/releases/download/v0.12.0/safe-browser-v0.12.0-win-x64.zip",
+	"hash": "189cc122ea30738d14725f056a3538439ebb5431396dd5062267f25ae927f9e1",
+	"bin": "SAFE Browser.exe",
+	"shortcuts": [
+		[
+			"SAFE Browser.exe",
+			"SAFE Browser"
+		]
+	],
+	"installer": {
```

Remove installer and replace it with:

```suggestion
    "extract_dir": "safe-browser-v0.12.0-win-x64",
```